docs/dev/testing.rst

```diff
@@ -108,11 +77,16 @@ datastore instead by using our custom ``--tmp-data`` with ``pytest``:
 -------------------------------------------------------------------------------
 Running pytest Directly
 -------------------------------------------------------------------------------
-Running tests directly with ``pytest`` gives you the ability to run only tests from a
-particular test module or even a single individual test case. It's also faster because
-there's no testing environment to set up. Instead, it just uses your Python environment
-which should be the ``pudl-dev`` conda environment discussed in :doc:`/dev/dev_setup`.
-This is convenient if you're debugging something specific or developing new test cases.
+Running tests directly with ``pytest`` gives you the ability to run only
+tests from a particular test module or even a single individual test case.
+It's also faster because there's no testing environment to set up. Instead,
```

Comment:
No longer true since we stopped using Tox, right?

Reply:
I guess that's right. Will update.